Daytime-Nighttime-Interactions/fault-reporting.rst

```diff
@@ -24,118 +24,105 @@
-
-- This is an example bullet of a post-condition (Telescope azimuth is 0 degrees.)
-- This is another example of a post-condition (This procedure leaves the telescope with the E-stop activated.)
-
-.. _Title-of-Procedure-Procedure-Steps:
+Upon navigating to the `(OBS) <https://jira.lsstcorp.org/projects/OBS/issues/OBS-4?filter=allopenissues>`__ project in JIRA, click the "create" option on the right side of the top tool-bar.
 
-Procedure Steps
-^^^^^^^^^^^^^^^
+.. figure:: ./_static/Fault_report_example_page_1.png
+    :name: Fault-report-example-page-1
+    :target: http://target.link/url
 
-.. todo::
-   Include utilization of admonishments (caution, warning, etc.)
+    Screenshot of an example fault report. 
+- Project field: The reporter should ensure that the OBS project is selected to include all things affecting nighttime operations.
 
-.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
-.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
-.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+- Issue type: A "problem" issue type usually refers to a hardware issue. A "bug" issue type typically refers to a software issue. 
+  If unsure, select "problem."
 
-This paragraph describes some general guidance for this procedure. This paragraph is optional depending on its usefulness and the complexity of the procedure.
+- Summary: Describe the problem in one phrase. Be as clear and succinct as possible.
 
-In the case of a very straightforward procedure, a simple numbered list could be used, like as follows:
+- Urgent: IMPORTANT. This field is crucial to allocate time to solve a problem. 
+  If this fault obstructs observing at night, data collection, or endangers equipment,
+  toggle this flag and alert night time staff as soon as possible. 
 
-.. warning::
-    For this example, this step is critical.
+- Time lost (hr): More details about calculating time lost due to a fault are in the :ref:`fault-reporting-Guidelines-for-calculating-time-loss` section. 
+  Time loss is reported in the 0.1 decimal hour.
 
-.. _Title-of-Procedure-Critical-Step-1:
+- Components: Be as accurate as possible to select the correct component - i.e. software, hardware: M2, etc. 
+  If the component does not exist, contact Alysha Shugart and they will add it to the list.
```

Comment:
Consider using `|author|` instead of your name explicitly, and update line 11 so it reads:

```
.. |author| replace:: *Name-of-Primary-Author*
```

e.g.;

```
If the component does not exist, contact |author| and they will add it to the list.
```